test/fixtures/semantic-html5-scenarios/section-with-chapters.adoc

```diff
@@ -0,0 +1,13 @@
+:doctype: book
+:sectnums:
+:noheader:
+
+= Chapter One
```

Comment:
This document is also not well-formed. What is marked as "Chapter One" is, in fact, the document title and parts are below chapters.